bin/ColorScript/Sources/ColorScript/main.swift

```diff
@@ -0,0 +1,10 @@
+import Foundation
+import ColorScriptCore
+
+let colorScript = ColorScript()
+
+do {
+  try colorScript.run()
+} catch {
+  print("Error")
```

Comment:
Can we add another case to the `ColorScriptError` enum to capture this error and return a more descriptive error message?

Reply:
I think here we could capture the error from the catch block and print the error message, like:
```
catch (let error) {
  print("Error: \(error.localizedDescription)")
}
```
What do you think?

Reply:
We could also have `ColorScriptError` conform to `CustomStringConvertible` so we can provide our own description of various error types. In that case all you would have to do is:

```
print("Error: \(error)")
```